### DISCUSSION TITLE
General git request response

### DISCUSSION THREAD

**Discussion by ThanThanSwe** [community/community#138415] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

This is my general discussion about push/pull request in github.

**Comment by thanthanswe0310** [comment]
Here is my git pull/push response 1.

**Comment by thanthanswe0310** [comment]
This is my git push/pull response 2.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/138417#discussioncomment-10618052) for reference to why this discussion was closed and locked. Thanks!
